handbook/FileService-MinIO/FileService-MinIO.md

```diff
@@ -0,0 +1,57 @@
+# MinIO as storage provider for the FileService
+
+SystemLink FileService allows to configure and usethe Amazon S3 cloud storage instead of a file share. While NI officially addresses the Amazon S3 cloud storage, the FileService also works with MinIO which is a Amazon S3 compatible server-side software storage stack.
+
+The minimum required SystemLink version is 2020 R3.
+
+In order to use MinIO, there are two steps required. First, setting up the MinIO server and, second, configuring the FileService to use that server.
+
+## Running MinIO server
+
+In order to run the MinIO server, download the server from the [MinIO website](https://min.io/download). From the command line, run the following command to run the server:
+
+```bash
+minio server <PathToRootFolder>
+```
+
+After starting the MinIO server, connect to the endpoint that is printed out in the command line by using a web browser and create a bucket by using the `+` button in the right bottom corner. This is only required if you run the MinIO server for the first time.
+
+!!! note "Note"
+    - It is recommended to set non-default access key and secret key. You can do that by setting the environment variables `MINIO_ACCESS_KEY` and `MINIO_SECRET_KEY`.
+    - There's no bucket region necessary for using MinIO, but you have to apply a bucket region for the FileService to work. Just use `us-east-1`.
+    - Any of the three endpoints in the first line of the minio server output are usable as `S3BackEndSecretKey` (for localhost given your client is on the same machine as the MinIO server).
+
+Detailed information on how to run the server, follow instructions from the [MinIO Quickstart guide](https://docs.min.io/docs/minio-quickstart-guide.html).
+
+## Configuring FileService
+
+Follow the instruction from [the documentation on uploading files to S3](https://www.ni.com/documentation/de/systemlink/latest/data/uploading-files-to-amazon-s3) to configure the FileService.
+
+Additionally to that, there are another two undocumented settings required only when using MinIO:
```

Comment:
I don't think we need to state they are undocumented since we documenting them here :-). Perhaps use "additional settings" rather than "undocumented settings"